packages/client/webapp/src/utils/TA2/stratify.ts

```diff
@@ -0,0 +1,25 @@
+// Providing the ID of 3 Models (model A, model B, and the type Model)
```

Comment:
Can you move this file to `/services/` instead of `/utils/` please? Dan mentioned he'd like to split it so that "services" is anything specific to this project or anything that depends on our APIs, and "utils" are general purpose math or DOM or data structure functions that could be copy-pasted to future Uncharted projects.

Also would be good to establish the naming convention of calling it `stratification-service.ts` or `model-transform-service.ts`

Reply:
Would you prefer `/services/TA2` or` /services`?

Reply:
Maybe `/services/models/`